schema.md

```diff
@@ -762,3 +769,23 @@ Based on that, added some new reference types.
 My plan is to rewrite the italicized paragraph at the top of the doc on Monday
 and then share the link publicly to gather more feedback from groups that have
 not yet seen it.
+
+## Status - 2021-08-05
+
+The biggest change to the schema is our decision to support multiple packages
+and ecosystems per entry. This is a reversal of our decision back in April (see
+"Status - 2021-04-23" for our rationale).
+
+This is primarily in the interests of supporting better interoperability with
+other vulnerability schemas, such as the [CVE JSON
+schema](https://github.com/CVEProject/cve-schema),
+where multiple packages are supported in a single entry. We've
+also been suggesting changes to the CVE schema  for better alignment
+([1](https://github.com/CVEProject/cve-schema/issues/86),
+ [2](https://github.com/CVEProject/cve-schema/issues/87),
+ [3](https://github.com/CVEProject/cve-schema/issues/88),
+ [4](https://github.com/CVEProject/cve-schema/issues/89)).
+
+This is a breaking change, but we hope to make migration easier by renaming the
+"affects" field to "affected" to allow existing consumers and producers of this
+data to more easily handle old and new versions of entries.
```

Comment:
It may be worth a comment like "We are still gathering experience about this new arrangement, and it may change again in the near future."

Reply:
Sure, added. But I'd rather we avoid future breaking changes unless absolutely necessary from this point. We're at a point with many consumers that expect stability and every breaking change (especially one as substantial as this) is quite painful.

Reply:
Then perhaps we should wait a bit on committing this until we get more experience from Go and OSI or anyone else?

Reply:
yes i don't think we should even make this change before consulting with the current consumers like Go, OSI, Rust. Can we get their opinion. is this a big usecase ?